code/modules/mob/living/simple_animal/borer/borer_powers.dm

```diff
@@ -239,32 +251,53 @@
 		return
 
 	if(world.time - used_dominate < 150)
-		to_chat(src, "You cannot use that ability again so soon.")
+		to_chat(src, SPAN_WARNING("You cannot use that ability again so soon."))
+		return
+
+	if(is_ventcrawling)
+		to_chat(src, SPAN_WARNING("You cannot use that ability while in vent."))
+		return
+
+	if(chemicals < 10)
+		to_chat(src, SPAN_WARNING("You don't have enough chemicals!"))
 		return
 
 	var/list/choices = list()
-	for(var/mob/living/carbon/C in view(3, get_turf(src)))
+	for(var/mob/living/carbon/C in view(1, get_turf(src)))
 		if(C == host || C.stat == DEAD)
 			continue
 
 		choices += C
 
-	if(world.time - used_dominate < 150)
-		to_chat(src, "You cannot use that ability again so soon.")
+	if(!choices)
+		to_chat(src, SPAN_WARNING("No available creatures found in your radius."))
 		return
 
 	var/mob/living/carbon/M = input(src,"Who do you wish to dominate?") in null|choices
 
-	if(!M || !(M in view(3, get_turf(src)))) return
+	if(world.time - used_dominate < 150)
+		to_chat(src, SPAN_WARNING("You cannot use that ability again so soon."))
+		return
+
+	if(!M || !(M in view(1, get_turf(src)))) return
+
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(istype(H.wear_suit, /obj/item/clothing/suit/space))
+			to_chat(src, SPAN_WARNING("You cannot use that ability on someone, who wear a space suit."))
+			return
 
 	if(M.has_brain_worms())
-		to_chat(src, "You cannot infest someone who is already infested!")
+		to_chat(src, "You cannot palyze someone who is already infested!")
```

Comment:
`to_chat(src, "You cannot palyze someone who is already infested!")`

Don't believe palyze is a word

Reply:
Paralyze is proper spelling